code/game/objects/items/holy_weapons.dm

```diff
@@ -11,6 +11,8 @@
 	throwforce = 10
 	w_class = WEIGHT_CLASS_TINY
 	var/reskinned = FALSE
+	var/clockwork_desc = "A rod of Obsidian, a material known for corroding the Justiciar's creations."
```

Comment:
Does this actually do anything? You just added a variable there, but didn't do anything with it.